mkdocs_awesome_pages_plugin/tests/navigation/test_order_and_sort.py

```diff
@@ -0,0 +1,279 @@
+from .base import NavigationTestCase
+from ...meta import Meta, MetaNavItem, MetaNavRestItem
+
+
+class TestOrderAndSort(NavigationTestCase):
+    def test_asc(self):
+        navigation = self.createAwesomeNavigation(
+            [self.page("1"), self.page("2"), self.page("3"), Meta(order=Meta.ORDER_ASC)]
+        )
+
+        self.assertNavigationEqual(navigation.items, [self.page("1"), self.page("2"), self.page("3")])
+        self.assertValidNavigation(navigation.to_mkdocs())
+
+    def test_asc_natural(self):
+        navigation = self.createAwesomeNavigation(
+            [
+                self.page("100."),
```

Comment:
What's up with the `.` suffix in the page names? Does `"100"` alone not work?

Reply:
It runs either way. I was looking to mimick where people had pages that were numbered. I'll add a test for sanity without the `.`.